### PR TITLE
Pick up custom config files

### PR DESCRIPTION
**Pull request by ntadimeti**
Code now picks up custom config files. Copy paste error. Verifyong tutorial 1 to make sure it works as expected.

### CODE REVIEW

**Comment by ntadimeti** [comment]
rerun tests

File: configs/infer_config.yaml [hunk 1]
```diff
@@ -40,3 +40,7 @@ batches_per_worker: 16
 result_fname: inferred
 gen_bigwig: True
 deletebg: False
+
+#Eval options
+best_metric_choice: "AUROC"
```
Comment (avantikalal) [inline configs/infer_config.yaml:45]: Why do we need best_metric during inference?

Comment (ntadimeti) [reply configs/infer_config.yaml:45]: We need for evaluation. Not for inference. `python main.py eval` needs it. This is in addition to all the options required by infer. It didn't make sense to have yet another config file, so I added these options in the infer_config.

**Comment by ntadimeti** [comment]
rerun tests

**Pull request merged**